frontend/src/metabase/home/components/EmbedMinimalHomepage/EmbedMinimalHomepage.tsx

```diff
@@ -0,0 +1,73 @@
+import { jt, t } from "ttag";
+import { Button, Card, Flex, Icon, Stack, Text, Title } from "metabase/ui";
+import Link from "metabase/core/components/Link";
+import ExternalLink from "metabase/core/components/ExternalLink";
+import { getDocsUrl } from "metabase/selectors/settings";
+import { useSelector } from "metabase/lib/redux";
+import { removeShowEmbedHomepageFlag } from "./util";
+import { NumberedList } from "./EmbedMinimalHomepage.styled";
+
+type EmbedMinimalHomepageProps = {
+  onDismiss: () => void;
+};
+
+export const EmbedMinimalHomepage = ({
+  onDismiss,
+}: EmbedMinimalHomepageProps) => {
+  const learnMoreUrl = useSelector(s =>
```

Comment:
Please try to refrain from using abbreviated names, and use the full name for clarity instead.

```suggestion
  const learnMoreUrl = useSelector(state =>
```

Reply:
👍 15564908a4185a238c0c1ccdd1d857a34a6501f8